.github/workflows/tox.yml

```diff
@@ -82,7 +82,7 @@ jobs:
     runs-on: ubuntu-24.04
     needs:
       - tox
-    # if: github.ref == 'refs/heads/main'
+    if: github.ref == 'refs/heads/main'
     # github.event_name == 'release' && github.event.action == 'published'
```

Comment:
Also remove the commented old condition but add a comment like explaining the publishing can only happen on push events happening on main branch, as pull-requests would not have such permissions.

Reply:
Done.